androidApp/src/main/java/com/mbta/tid/mbta_app/android/routeDetails/CollapsableStopList.kt

```diff
@@ -0,0 +1,157 @@
+                        // TODO: Content description
+                        .padding(horizontal = 10.dp)
+                        .defaultMinSize(minHeight = 48.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    AnimatedContent(
+                        stopsExpanded,
+                        transitionSpec = {
+                            fadeIn(animationSpec = tween(500)) togetherWith
+                                fadeOut(animationSpec = tween(500))
+                        },
+                    ) {
+                        Row(
+                            verticalAlignment = Alignment.CenterVertically,
+                            horizontalArrangement = Arrangement.Center,
+                            modifier = Modifier.padding(start = 22.dp).width(20.dp),
+                        ) {
+                            if (it) {
+                                Icon(
+                                    painterResource(R.drawable.fa_caret_right),
+                                    contentDescription = null,
+                                    modifier = Modifier.size(12.dp).rotate(90f),
+                                    tint = colorResource(R.color.deemphasized),
+                                )
+                            } else {
+                                Icon(
+                                    painterResource(R.drawable.fa_caret_right),
+                                    contentDescription = null,
+                                    modifier = Modifier.size(12.dp),
+                                    tint = colorResource(R.color.deemphasized),
+                                )
+                            }
+                        }
+                    }
+                    Column(
+                        modifier =
+                            Modifier.weight(1f)
+                                .padding(vertical = 12.dp)
+                                .padding(horizontal = 16.dp),
+                        verticalArrangement = Arrangement.spacedBy(4.dp),
+                    ) {
+                        Text(
+                            AnnotatedString.fromHtml(
+                                stringResource(R.string.less_common_stops, segment.stops.size)
+                            ),
+                            color = colorResource(R.color.text),
+                            style = Typography.body,
+                        )
+                        Text(
+                            stringResource(R.string.only_served_at_certain_times),
+                            color = colorResource(R.color.deemphasized),
+                            style = Typography.footnote,
+                        )
+                    }
+                }
+
+                if (!isLastSegment) {
+                    HaloSeparator()
+                }
+            }
+
+            if (stopsExpanded) {
+                segment.stops.map { stop ->
```

Comment:
`.map` with a lambda returning `Unit` just builds a list of `Unit`s for no reason. I think just a `for` loop would make more sense.